rules.md

```diff
@@ -66,3 +66,221 @@ and fail unexpectedly when supplied extra long strings.
 ## H001: Base path can be extracted
 
 If all paths start with the same prefix then it would be cleaner to extract that into the basePath rather than repeating for each path.
+
+##
+
+
+As well as the additional rules coming from Zally, there are also additional Boat Quay rules, some of which are Backbase 
+specific but could be transferable. These additional rules, are documented below.    
+##
+
+## B012: Use Well Understood HTTP Status Codes
```

Comment:
Perhaps order the rules in ascending order rather than descending order (to make consistent with how the other rules are ordered)